src/components/formFields/input.tsx

```diff
@@ -62,6 +64,7 @@ export const InputEmail: React.FC<IInputProps & IReactHookFormProps> = ({
     {...{ defaultValue, disabled, placeholder }}
     {...register(name, { ...validation })}
     invalid={errors[name]}
+    icon={<EmailIcon />}
```

Comment:
If we add a hardcoded icon here, we're forcing users who implement this `InputEmail` component to have this specific icon.

I can imagine this breaks the following use cases: (1) users who want to use the component _without_ any icon and (2) users who want to use this component with their own icon.

Therefore, I think adding an Icon to the `InputEmail` component is fine, however, it should be implemented in the same way we do at (e.g.) `InputText`.